lib/rex/compat.rb

```diff
@@ -198,7 +198,7 @@ def self.open_webrtc_browser(url='http://google.com/')
   else
     # If DISPLAY is not set, most unix graphical tools will not work.
     if ENV['PATH'] && ENV['DISPLAY']
-      ['google-chrome', 'chrome', 'chromium', 'firefox' , 'firefox', 'opera'].each do |browser|
+      ['xdg-open', 'google-chrome', 'chrome', 'chromium', 'firefox' , 'firefox', 'opera'].each do |browser|
```

Comment:
I'm not sure if this fix is cromulent, as files won't always open up in a browser.

i.e. `xdg-open foo.txt` opens in my text editor, versus `firefox foo.txt` which opens in the browser